Bottom: 45ad339781118529fc9268ba4314c9176e04466b
Top:    0fb4682d8c8b501e380e109fda0c539ee3c92ec5
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-07 22:19:23 -0400

Refresh of temp

---

diff --git a/sound/soc/fsl/mpc5200_dma.c b/sound/soc/fsl/mpc5200_dma.c
index 763be11..5704a56 100644
--- a/sound/soc/fsl/mpc5200_dma.c
+++ b/sound/soc/fsl/mpc5200_dma.c
@@ -35,8 +35,6 @@ MODULE_AUTHOR("Grant Likely <grant.likely@secretlab.ca>");
 MODULE_DESCRIPTION("Freescale MPC5200 PSC in DMA mode ASoC Driver");
 MODULE_LICENSE("GPL");
 
-static int stop = 0;
-
 /*
  * Interrupt handlers
  */
@@ -209,7 +207,6 @@ static int psc_dma_trigger(struct snd_pcm_substream *substream, int cmd)
 
 		out_8(&regs->command, MPC52xx_PSC_RST_ERR_STAT);
 
-		stop = 0;
 		spin_unlock_irqrestore(&psc_dma->lock, flags);
 
 		break;
